raft/node.go

```diff
@@ -60,6 +60,12 @@ type Ready struct {
 	// HardState will be equal to empty state if there is no update.
 	pb.HardState
 
+	// The node can serve linearizable read requests locally when its applied index is
+	// greater than the index in ReadState.
+	// Note that the readState will be returned when raft receives msgReadIndex The returned
```

Comment:
Note that the readState will be returned when raft receives msgReadIndex.
The returned is only valid for the request that requested to read.